src/ctapipe/image/muon/fitting.py

```diff
@@ -50,22 +66,79 @@ def kundu_chaudhuri_circle_fit(x, y, weights):
         np.sum(weights * ((center_x - x) ** 2 + (center_y - y) ** 2)) / weights_sum
     )
 
-    return radius, center_x, center_y
+    if nan_errors_flag:
```

Comment:
Please explain why we would want to switch off error computation.

In case we really do want that, i'd name the variable `compute_errors={True,False}` to explain better what it is about.